External/FEXCore/Source/Interface/IR/PassManager.cpp

```diff
@@ -30,7 +30,7 @@ void PassManager::AddDefaultPasses(FEXCore::Context::Context *ctx, bool InlineCo
 
     InsertPass(CreateDeadStoreElimination());
     InsertPass(CreatePassDeadCodeElimination());
-    InsertPass(CreateConstProp(InlineConstants));
+    InsertPass(CreateConstProp(InlineConstants, ctx->HostFeatures.SupportsRCPC2));
```

Comment:
This should be on for x86, so we test that TSO memops with offsets go through the pipeline correctly.

Reply:
Enabled on x86 as well. Resolved.